Library/Formula/ice.rb

```diff
@@ -8,7 +8,13 @@ class Ice < Formula
   option 'doc', 'Install documentation'
   option 'demo', 'Build demos'
 
-  depends_on 'berkeley-db'
+  resource "berkeley-db" do
+    url 'http://download.oracle.com/berkeley-db/db-5.3.28.tar.gz'
+    sha1 'fa3f8a41ad5101f43d08bc0efb6241c9b6fc1ae9'
```

Comment:
@mikemcquaid What's the goal of using double quotes versus simple quotes ? There is no special interpolation in those current strings